libs/vertexai/tests/integration_tests/test_tools.py

```diff
@@ -45,231 +34,3 @@ def parse_result(
 
     def parse(self, text: str) -> Union[AgentAction, AgentFinish]:
         raise ValueError("Can only parse messages")
-
-
-@pytest.mark.xfail(reason="investigating")
```

Comment:
why do you think we should remve these tests?

Reply:
all tool calling abilities are now tested by the standard tests so they're redundant